Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,413 @@
+    , DstGeoCountry = iff(NetworkDirection == "Outbound", sosCountry, "")
+    , SrcAppName = iff(NetworkDirection in ("Inbound", "Local", "NA"), coalesce(appcat, appName), "")
+    , DstAppName = iff(NetworkDirection in ("Outbound", "Local", "NA"), coalesce(appcat, appName), "")
+    , SrcAppId = iff(NetworkDirection in ("Inbound", "Local", "NA"), sid, "")
+    , DstAppId = iff(NetworkDirection in ("Outbound", "Local", "NA"), sid, "")
+  | extend
+    SrcAppType = case(isempty(SrcAppName), ""
+                      , SrcAppName contains "\'General " or SrcAppName contains "\'Service ", "Service", "Other")
+    , DstAppType = case(isempty(DstAppName), ""
+                      , DstAppName contains "\'General " or DstAppName contains "\'Service ", "Service", "Other")
+  | project-rename
+    sosReceivedPackets = DeviceCustomNumber1Label // DeviceCustomNumberXLabel (cnXLabel=)
+    , sosSentPackets = DeviceCustomNumber2Label // DeviceCustomNumberXLabel (cnXLabel=)
+    , sosConnectionDuration = DeviceCustomNumber3Label
+    , sosUser = susr // Logged-in username associated with the log event.
+    , sosAppRulePolicyId = af_polid // App Rule Policy ID.
+    , sosAppRulePolicyName = af_policy // App Rule Policy Name.
+    , sosAppRuleService = af_service // App Rule Service Name.
+    , sosAppRuleType = af_type // App Rule Policy Type.
+    , sosAppRuleObject = af_object // App Rule Object Name.
+    , sosAppRuleObjectContent = contentObject // App Rule Object Content.
+    , sosAppRuleAction = af_action
+    , sosSourceIPv6Address = srcV6
+    , sosDestinationIPv6Address = dstV6
+    , sosAppFullString = appcat // The full "<CATEGORY> <APPNAME> -- <SIGNAME>" string.
+    , sosAppIDNumber = app // Numeric Application ID. Not the same as "ApplicationProtocol".
+    , sosAppID = appid // Application ID from App Control
+    , sosAppCategoryID = catid // Application Category ID
+    , sosAppSignatureID = sid // Application Signature ID
+    , sosIPSCategoryName = ipscat // IPS Category Name
+    , sosAntiSpywareCategory = spycat // Anti-Spyware Category
+    , sosURLPathName = arg // URL. Represents the URL path name.
+    , sosFileIdentifier = fileid // File hash or URL
+    , sosDPIInspectedFlow = dpi // Indicates a flow was inspected by DPI. Applies only to Connection Closed messages.
+    , DstNatPortNumber = dnpt
+    , SrcNatPortNumber = snpt
+    , sosBladeID = bid // Blade ID
+    , sosUUID = uuid
+    , sosHttpRequestMethod = HttpRequestMethod
+    , sosFileName = FileName
+    , DvcOriginalAction = fw_action
+  | extend
+    ThreatName = coalesce(sosASWSignatureName, sosGAVSignatureName, sosIPSSignatureName, "")
+    , ThreatId = coalesce(sosAppSignatureID, "")
+    , ThreatCategory = coalesce(sosIPSCategoryName, sosAntiSpywareCategory, "")
+    , DstNatPortNumber = toint(DstNatPortNumber)
+    , SrcNatPortNumber = toint(SrcNatPortNumber)
+  | extend AdditionalFields = bag_pack(
+    "ReceivedPackets", sosReceivedPackets

```

Comment:
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/7ad5fb05-ff8d-48d5-a019-a5a5795f0418)

Reply:
@jaimeesc 
did you add this mapping? unable to locate in parser

Reply:
@jaimeesc my bad, looking at wrong parser. I can see you have mapped it.